### ISSUE TITLE
What files are required for a translation project?

### ISSUE THREAD

**Issue by carukc** [rtfd/readthedocs.org#1314]
The ReadTheDocs documentation recommends a project for each translation.  What is the minimum set of files required for a translation project?

I assumed that it was:

readme.rst
setup.py
docs/
docs/config.py
docs/locale/<your target language>/LC_MESSAGES/...
docs/.tx/config (so that the tx client knows where to pull from)

but... docs generate but PDFs don't.

Thanks
Chris

**Comment by gregmuellegger** [comment]
What project are you working on? It would be good if I could have a look at the build output that is causing those issues.

**Comment by carukc** [comment]
Hi.  Thanks for responding.... When I asked the question it was not clear to me that the second project that is described the the RTD documentation is in face just a different RTD project that utilizes the same git project.  Once I pointed the new (French) RTD project at the main code repository (that included the original English text and the translated .po files, everything worked.  I also did not realize that the 'locale directory needed to be added to the main git repo but after looking at the RTD project itself I figured this out.

I owe this project a few mods to the documentation that I hope will help clarify this.  I've read through the docs on collaborating and will fork the project and create the pull requests this week.
